### ISSUE TITLE
Problems creating apps with SSL certificate

### ISSUE THREAD

**Issue by benjamino21** [frikky/Shuffle#57]
Hi Frederik, i'm trying to create a QRadar app but I can't connect to my server because it has an SSL certificate and when i try, it gives me Authentication Error because of SSL. Is it posible to create a box to enable/disable SSL verification or a way to introduce my SSL file? I've tried to enter -k argument into curl request when I create actions but it doesn't accept it.

Thank you very much

**Comment by frikky** [comment]
Hey! I read your email a bit back and thought you made it manually, but this totally makes sense! I'll add it as an optional option to every action if no URL is specified for now. I'll get it done ASAP :)

**Comment by frikky** [comment]
Hey again, for now I won't be fixing a way to upload a file from the GUI into the Docker container, but I enabled the ability to disable SSL verification if the URL field is empty. This is to enable the OpenAPI specification to work for other people than just yourself without any modification requirements.

![image](https://user-images.githubusercontent.com/5719530/84570084-20c40c80-ad8b-11ea-9aa4-67a489ee019b.png)

If you really need to add your own certificate, it's a bit more tricky as you're using pre-built Docker containers and the Dockerfile used for app building is 
Here's your options: 
* Modify the current running build. This will have to be re-done every time you restart the shuffle-backend Docker container:
1. Modify shuffle/backend/app_gen/openapi/baseline/Dockerfile with your certificates
2. Copy the new dockerfile into Shuffle:
```
docker cp Dockerfile shuffle-backend:/app_gen/openapi/baseline/Dockerfile 
```

* Rebuild the entire thing:
1. Add your certificates to the build process in shuffle/backend/app_gen/openapi/baseline/Dockerfile
2. Rebuild the backend (uncomment build for backend in docker-compose.yml https://github.com/frikky/Shuffle/blob/master/docker-compose.yml#L29), then run:
```
docker-compose down
docker-compose up --build
```

I'm aware none of these are ideal, so I created an issue to override this with remote locations later on: #58
